app/src/fdroid/java/openfoodfacts/github/scrachx/openfood/features/scan/ContinuousScanActivity.kt

```diff
@@ -646,16 +651,11 @@ class ContinuousScanActivity : AppCompatActivity() {
                     }
                     R.id.troubleScanning -> {
                         hideAllViews()
-                        hintBarcodeDisp?.dispose()
-
+                        hintBarcodeDisp!!.dispose()
                         binding.quickView.setOnClickListener(null)
                         binding.quickViewSearchByBarcode.text = null
                         binding.quickViewSearchByBarcode.visibility = View.VISIBLE
-                        binding.quickView.visibility = View.INVISIBLE
-                        quickViewBehavior.state = BottomSheetBehavior.STATE_EXPANDED
-                        commonDisp.add(Completable.timer(500, TimeUnit.MILLISECONDS)
```

Comment:
Why are we removing this?

Reply:
1. Not sure of its utility.
2. Trouble scanning is working after changing this